schema/partner.js

```diff
@@ -86,6 +91,24 @@ const PartnerType = new GraphQLObjectType({
           }));
         },
       },
+      artworks: {
+        type: new GraphQLList(Artwork.type),
+        args: {
+          size: {
+            type: GraphQLInt,
+          },
+          sort: ArtworkSorts,
+          exclude: {
+            type: new GraphQLList(GraphQLString),
+          },
+        },
+        resolve: ({ id }, options) => {
+          return gravity(`filter/artworks`, assign({}, options, {
```

Comment:
No, just request `/partner/:id/artworks` with the sort? If it doesn't support the desired sort, we can probably add it.